libvmi/driver/xen/xen_events.c

```diff
@@ -376,7 +376,7 @@ status_t process_unhandled_mem(vmi_instance_t vmi,
     mem_access_event_t event = { 0 };
     event.physical_address = page->key << 12;
     event.npages = 1;
-    xen_set_mem_access(vmi, &event, VMI_MEMACCESS_N);
+    xen_set_mem_access(vmi, &event, VMI_MEMACCESS_N, req->altp2m_idx);
```

Comment:
Is `req->altp2m_idx` guaranteed to be 0 if VM_EVENT_ALTERNATE_P2M is cleared?  "may be used" (from http://xenbits.xen.org/gitweb/?p=xen.git;a=blob;f=xen/include/public/vm_event.h;h=9270d52592af52de51cfde07bc492a504412edb2;hb=HEAD#l241) sounds underspecified to me.